### ISSUE TITLE
Java

### ISSUE THREAD

**Issue by JoniWebb** [OpenLiveWriter/OpenLiveWriter#417]
I am trying to include html code into my blogger blog and it jst won't take it.  
when i paste through paste html it says java isn't activated.
what am i doing wrong? do you know?  much thanks in advance and sorry this is so stupid sounding!

**Comment by ScottIsAFool** [comment]
Can you post a screenshot of what you're seeing please?

**Comment by JoniWebb** [comment]
here you go - you can see the message at the bottom of the post.
thanks!




-----Original Message-----

**Comment by JoniWebb** [comment]
sorry, here it is.  



-----Original Message-----

**Comment by ScottIsAFool** [comment]
There's still no screenshot, you need to attach it on github rather than in an email reply.

**Comment by JoniWebb** [comment]
![image](https://cloud.githubusercontent.com/assets/16286301/13555904/304c9464-e392-11e5-829e-95501d6a344a.png)


here it is I HOPE!!!

**Comment by kathweaver** [comment]
What html code were you posting?

**Comment by JoniWebb** [comment]
it's widgets from Restyleme.


here is an example of th code i was trying to insert into my blog.  once you put it in, a picture should pop up.  Instead I kept getting that message.   This is designed to work in the blog story itself.


<div class="shopthepost-widget" data-widget-id="1451863"><script type="text/javascript">!function(d,s,id){var e, p = /^http:/.test(d.location) ? 'http' : 'https';if(!d.getElementById(id)) {e = d.createElement(s);e.id = id;e.src = p + '://' + 'widgets.rewardstyle.com' + '/js/shopthepost.js';d.body.appendChild(e);}if(typeof window.__stp === 'object') if(d.readyState === 'complete') {window.__stp.init();}}(document, 'script', 'shopthepost-script');</script><div class="rs-adblock"><img src="//assets.rewardstyle.com/images/search/350.gif" onerror="this.parentNode.innerHTML='Disable your ad blocking software to view this content.'" style="width: 15px; height: 15px;" /><noscript>JavaScript is currently disabled in this browser. Reactivate it to view this content.</noscript></div></div>




Joni



-----Original Message-----

**Comment by Laronski** [comment]
Does this work if it's pasted directly into the Blogger edit/compose page? Without using Open Live Writer?

**Comment by JoniWebb** [comment]
it doesn't work anywhere.  i tried it on Explorer Live as opposed to Blogger.
it still gives me the java error.  
checiking with rewardstyle.  see what they say.


thanks.
j

-----Original Message-----

**Comment by Laronski** [comment]
In my opinion... If the code doesn't work in Open Live Writer or pasted directly into the compose editor in Blogger [browser editor]; I don't think the problem is with Open Live Writer. I think the problem lies with the code or blogger allowing it.

If you can get the code to work without using Open Live Writer at all; it'd be worth looking at closer.

**Comment by JoniWebb** [comment]
i agree.  I don't think it's on your end I think it's on Restyleme's end.   So, I'm just out of luck.
Oh well, thanks for looking at it.


Joni



-----Original Message-----

**Comment by shanselman** [comment]
Thanks for reporting this and for helping Open Live Writer over the years. We are doing an issue-bankruptcy cleanup so we can make the repository maintainable again and focus on current, actionable work. This older issue is being closed as part of that cleanup, not because it was unimportant. If this still reproduces in the current code or is still relevant to the revitalization work, please open a fresh issue with current details and we will triage it going forward.
